app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsNav.kt

```diff
@@ -28,11 +27,8 @@ fun NavController.restoreMnemonics(
 @Serializable
 sealed interface RestoreMnemonicsArgs {
     @Serializable
-    data class RestoreProfile(val backupType: BackupType) : RestoreMnemonicsArgs
-
-    @Serializable
-    data class RestoreSpecificMnemonic(
-        val factorSourceId: HexCoded32Bytes,
+    data class RestoreProfile(
```

Comment:
there is no need for sealed interface here anymore, I suggrest removing it and just having data class called `RestoreMnemonicsArgs` that will contain properties that are now inside `RestoreProfile`

Reply:
Will do in next PR Im doing, good point @jakub-rdx